skbio/sequence/_rna.py

```diff
@@ -136,6 +136,72 @@ def _motifs(self):
         return _motifs
 
     @stable(as_of="0.4.0")
```

Comment:
I agree that this is stable, but make it "0.4.0-dev". When we release we will figure out what we are doing with the versions.